src/Integrations/Vue/Collection/createVuexCollectionFromState.js

```diff
@@ -0,0 +1,19 @@
+import Collection from '../../../Core/Resource/Collection';
+
+export default function createVuexCollectionFromState(name, state, repository, store) {
+  const collection = Collection.fromState(name, state, repository);
+
+  const { load } = collection;
+  collection.load = () => {
```

Comment:
Ik begrijp het overschrijven van de load method maar ik ben er niet helemaal zeker van.
Op de een of andere manier voelt het nog steeds ok om de load te behouden zoals deze is en een andere method toe te voegen die load met vuex store doet. De method naam die ik bedacht had was echter niet heel goed (loadItems). Misschien toch maar gewoon eerst zo laten.